src/platform/packages/shared/kbn-apm-synthtrace/src/cli/utils/get_clients.ts

```diff
@@ -89,3 +89,13 @@ export async function getClients({
     esClient: options.client,
   };
 }
+
+export function cloneClients(clients: SynthtraceClients): SynthtraceClients {
```

Comment:
Clone is needed because each scenario has its bootstrap, which can mutate a client state.  Client state such as `pipeline` is per scenario

Reply:
The client mutation or the pipeline per scenario?